pkgs/applications/editors/rstudio/default.nix

```diff
@@ -61,18 +62,15 @@ let
   rsconnectSrc = fetchFromGitHub {
     owner = "rstudio";
     repo = "rsconnect";
-    rev = "e287b586e7da03105de3faa8774c63f08984eb3c";
-    sha256 = "sha256-ULyWdSgGPSAwMt0t4QPuzeUE6Bo6IJh+5BMgW1bFN+Y=";
+    rev = "db2bcba0d82698aca6bc6aff4f639cc531060b4b";
+    sha256 = "sha256-Lv3WBTb0oP0ibP3AB0v3+E+i2DJoADMZ6Ysfxnk3ZvE=";
   };
 
-  panmirrorModules = mkYarnModules rec {
-    inherit pname version;
-    packageJSON = ./package.json;
-    yarnLock = "${src}/src/gwt/panmirror/src/editor/yarn.lock";
-    offlineCache = fetchYarnDeps {
-      inherit yarnLock;
-      hash = "sha256-v05Up6VMlYlvgUYQVYo+YfpcsMohliNfMgyjq6QymCI=";
-    };
+  quartoSrc = fetchFromGitHub {
+    owner = "quarto-dev";
+    repo = "quarto";
+    rev = "a92906bbd004e3d8abc39b5f59346ca2a091d1e8";
```

Comment:
Can you double check this? I think this version of rstudio is mountain hydrangea, and the `rstudio-mountain-hydrangea` branch of `quarto` is currently [01345470a8f80becb1e128be24f5](https://github.com/quarto-dev/quarto/commit/01345470a8f80becb1e128be24f59d2c34fb3a85).

Reply:
I think it's "Elsbeth Geranium" , but I got the idea :)

Reply:
Because there are only 2 release branches, and the later one is mountain-hydrangea. They don't have one for "Elsbeth Geranium"

Reply:
Actually, the hydrangea release was _earlier_ than the current rev in the default.nix. So I looked a bit more into it and chose a rev that is close to the release date of rstudio. Probably it's not exactly what Posit is using, but close enough. Everything still works, except visual mode.